lib/executor/helpers.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"github.com/sirupsen/logrus"
 
+	"github.com/loadimpact/k6/js/common"
```

Comment:
you should not be importing `k6/js/*` anything from anywhere in `lib`. 

This has many reasons but in general we don't want to tie the `lib` functionality to a specific runner even if we only currently have `js` one and it is unlikely we will ever support others(and there are many other problems in the way of it)

Reply:
I moved `InterruptError` to `lib/errors`. 

done 75f8e87b